package.json

```diff
@@ -5,6 +5,10 @@
   "main": "lib/index.js",
   "license": "MIT",
   "author": "Jon Gold <jon.gold@airbnb.com> (http://jon.gold)",
+  "engines": {
```

Comment:
engines is advisory only, so it doesn't add much value; that said, this should work on node 6 and npm 3.